jekyll-watch.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "listen", "~> 3.0"
+  spec.add_runtime_dependency "listen", "~> 3.0", "< 3.1"
```

Comment:
I actually thought about that! How about: "XXX: Remove with Jekyll 4 or in 2017 when 2.1 goes EOL"?